test/functional/wallet_groups.py

```diff
@@ -64,6 +64,52 @@ def run_test(self):
         assert_approx(v[0], 0.2)
         assert_approx(v[1], 1.3, 0.0001)
 
+        # Test 'avoid partial if warranted, even if disabled'
```

Comment:
nit: use `self.log.info` instead of a comment.

Reply:
Sure, will tweak if I end up needing to rebase.